utils/keys.go

```diff
@@ -103,11 +104,45 @@ func ImportKeys(from io.Reader, to []Importer) error {
 		toWrite []byte
 	)
 	for block, rest := pem.Decode(data); block != nil; block, rest = pem.Decode(rest) {
+		if block.Headers["role"] == "" {
+			// no worries about if check as for GUN here because empty roles will get a role:notary.DefaultImportRole
+			block.Headers["role"] = role
+		}
+		if block.Headers["gun"] == "" {
```

Comment:
Non-blocking:  this has probably already been discussed and resolved, and I just completely forgot all the cases, but is it ok for the gun and path to mismatch?  e.g. gun to be `docker.io/hello/world` and path to be `/tuf_keys/docker.io/nothello/notworld/DEADBEEF0A...key`?  I know we don't really use the header GUN for anything right now, so it doesn't much matter - just wondering if it's worth trying to grab the GUN from the path first before falling back on no gun?